HousingSearchApi/V1/Gateways/GetPersonGateway.cs

```diff
@@ -1,21 +1,25 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using HousingSearchApi.V1.Domain;
+using Microsoft.Extensions.Logging;
 using Nest;
 
 namespace HousingSearchApi.V1.Gateways
 {
     public class GetPersonGateway : IGetPersonGateway
     {
         private readonly IElasticClient _esClient;
+        private readonly ILogger<GetPersonGateway> _logger;
 
-        public GetPersonGateway(IElasticClient esClient)
+        public GetPersonGateway(IElasticClient esClient, ILogger<GetPersonGateway> logger)
         {
             _esClient = esClient;
+            _logger = logger;
         }
 
         public Task<List<Person>> Search(SearchParameters parameters)
         {
+            _logger.LogDebug($"Returning results based on the following parameters {parameters}");
```

Comment:
You might need to take out this line. 
Logging the search parameters might mean you could be logging personal information (e.g. names) which we absolutely should not be doing. Just have a look to make sure that won't happen.

Reply:
Good Point! I've refactored this so that it only specified the log level